serializers/web/invoice.rb

```diff
@@ -43,7 +47,20 @@ def self.base(inv)
               name: resource["resource_name"],
               description: line_item["description"],
               duration: line_item["duration"].to_i,
-              cost: (line_item["cost"] < 0.001) ? "less than $0.001" : "$%0.03f" % line_item["cost"]
+              cost: line_item["cost"],
+              cost_humanized: humanized_cost(line_item["cost"])
+            }
+          end
+        end.group_by { _1[:description] }.flat_map do |description, line_items|
+          if line_items.count <= 5
+            line_items
+          else
+            {
+              name: "#{line_items.count} x #{description} (Aggregated)",
+              description: description,
+              duration: line_items.sum { _1[:duration] },
+              cost: line_items.sum { _1[:cost] },
```

Comment:
I meant putting (Aggregated) to duration and cost (because they are the ones which are aggregated), not to the name part.

Reply:
Yes, I got and tried it. However, applying it to the duration and cost columns disrupts their alignment. I haven't spent much time on this as this is a workaround.

Reply:
What do you think about `standard-4 Virtual Machine (Aggregated 13 Items)` instead of `13 x standard-4 Virtual Machine (Aggregated)`

Reply:
Both are fine. This was a minor comment. If it requires too much effort, feel free to do whatever is easiest.

Reply:
I added it to my backlog. I'll consider it further when I'm working on the UI.